AmplifyPlugins/Auth/AWSCognitoAuthPluginIntegrationTests/Support/AuthSessionHelper.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+@testable import AWSMobileClient
+@testable import Amplify
+import AWSCognitoAuthPlugin
+
+struct AuthSessionHelper {
+
+    static func invalidateSession(username: String) {
+        let bundleID = Bundle.main.bundleIdentifier
+        let keychain = AWSUICKeyChainStore(service: "\(bundleID!).\(AWSCognitoIdentityUserPool.self)")
+        let namespace = "\(AWSMobileClient.default().userPoolClient!.userPoolConfiguration.clientId).\(username)"
+        let key = "\(namespace).tokenExpiration"
```

Comment:
I don't like using production code to set up tests, but in this case, it seems like we should be using the exact same logic to make sure all session data is being removed. If we change the way session data is keyed and forget to update this method, we run the risk of hard-to-diagnose test failures.